nixos/modules/virtualisation/docker.nix

```
@@ -69,7 +69,7 @@ in
         type = types.bool;
         default = false;
         description = ''
-          **Deprecated**, please use virtualisation.containers.cdi.dynamic.nvidia.enable instead.
+          **Deprecated**, please use hardware.nvidia-container-toolkit.enable instead.
```

```suggestion
          **Deprecated**, please use {option}`hardware.nvidia-container-toolkit.enable` instead.
```

IIRC this is how you cross-reference options in the manual